### PR TITLE
mods: add page

### PR DESCRIPTION
**Pull request by CovertDuck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): v0.2.0**

### CODE REVIEW

**Review by maaslalani — APPROVED** [review]

**Review by Waples — APPROVED** [review]
lgtm

**Review by 4G3NT — APPROVED** [review]

File: pages/common/mods.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`mods -f "write a new section to this readme for a feature that sends you a free rabbit if you hit r"`
+
+- Organize your videos, in a markdown [f]ormat:
+
+`ls {{path/to/videos/}} | mods -f "organize these by decade and summarize"`
+
+- Read through raw HTML and summarize the contents, in a markdown [f]ormat:
+
+`curl "https://api.open-meteo.com/v1/forecast?latitude=29.00&longitude=-90.00&current_weather=true&hourly=temperature_2m,relativehumidity_2m,windspeed_10m" 2>/dev/null | mods -f "summarize this weather data for a human."`
```
Comment (kbdharun) [inline pages/common/mods.md:32]:
```suggestion
# mods

> AI for the command line, built for pipelines.
> More information: <https://github.com/charmbracelet/mods>.

- Ask for a generic question or request:

`mods "{{write me a poem about platypuses}}"`

- Open [s]ettings in your $EDITOR:

`mods -s`

- Ask for comments on your code, in a markdown [f]ormat:

`mods -f "{{what are your thoughts on improving this code?}}" < {{path/to/file}}`

- Ask for help with your documentation, in a markdown [f]ormat:

`mods -f "{{write a new section to this readme for a feature that sends you a free rabbit if you hit r}}"`

- Organize your videos, in a markdown [f]ormat:

`ls {{path/to/videos}} | mods -f "{{organize these by decade and summarize}}"`

- Read through raw HTML and summarize the contents, in a markdown [f]ormat:

`curl "{{https://api.open-meteo.com/v1/forecast?latitude=29.00&longitude=-90.00&current_weather=true&hourly=temperature_2m,relativehumidity_2m,windspeed_10m" 2>/dev/null}} | mods -f "{{summarize this weather data for a human.}}"`

- Display [h]elp and exit:

`mods -h`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

Suggestions/Feedback:

- We suggest adding `{{}}` to placeholders to be highlighted by our clients.
- We generally prefer to add the Help command to the last part of the page and use generic phrases like "Display help" as the command description to be consistent with all pages.

In the below/above suggestion, I have performed all the required changes for you to commit it easily.

**Comment by CovertDuck** [comment]
> Thanks for your contribution and welcome to tldr. I have some suggestions for this page.
> 
> Suggestions/Feedback:
> 
> * We suggest adding `{{}}` to placeholders to be highlighted by our clients.
> * We generally prefer to add the Help command to the last part of the page and use generic phrases like "Display help" as the command description to be consistent with all pages.
> 
> In the below/above suggestion, I have performed all the required changes for you to commit it easily.

My thanks for the feedback.
I committed these changes as requested. :)

File: pages/common/mods.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`mods -f "{{write a new section to this readme for a feature that sends you a free rabbit if you hit r}}"`
+
+- Organize your videos, in a markdown [f]ormat:
+
+`ls {{path/to/videos}} | mods -f "{{organize these by decade and summarize}}"`
+
+- Read through raw HTML and summarize the contents, in a markdown [f]ormat:
+
+`curl "{{https://api.open-meteo.com/v1/forecast?latitude=29.00&longitude=-90.00&current_weather=true&hourly=temperature_2m,relativehumidity_2m,windspeed_10m" 2>/dev/null}} | mods -f "{{summarize this weather data for a human.}}"`
```
Comment (acuteenvy) [inline pages/common/mods.md:28]: `/dev/null` won't work on Windows, it uses `NUL`.

Also, why redirect `stderr` in the first place? If there is no error, nothing gets printed to `stderr` in this example, and if there is, it won't work either way. This just suppresses error messages. 

```suggestion
- Read through raw HTML and summarize the contents, in markdown format:

`curl "{{https://api.open-meteo.com/v1/forecast?latitude=29.00&longitude=-90.00&current_weather=true&hourly=temperature_2m,relativehumidity_2m,windspeed_10m}}" | mods --format "{{summarize this weather data for a human}}"`
```

File: pages/common/mods.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display [h]elp and exit:
+
+`mods -h`
```
Comment (acuteenvy) [inline pages/common/mods.md:32]:
```suggestion
- Display help and exit:

`mods --help`
```

**Review by acuteenvy — COMMENTED** [review]
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**